port/win/env_win.cc

```diff
@@ -2071,8 +2077,13 @@ class WinEnv : public Env {
   std::vector<ThreadPool> thread_pools_;
   mutable std::mutex mu_;
   std::vector<std::thread> threads_to_join_;
+  static FnGetSystemTimePreciseAsFileTime GetSystemTimePreciseAsFileTime_;
+  static bool GetSystemTimePreciseAsFileTimeInitialized_;
 };
 
+FnGetSystemTimePreciseAsFileTime WinEnv::GetSystemTimePreciseAsFileTime_ = NULL;
+bool WinEnv::GetSystemTimePreciseAsFileTimeInitialized_ = false;
```

Comment:
Not necessary and not thread-safe if it was necessary. The _ctor executes only once under std::once() protection.